src/main/java/org/apache/commons/lang3/SortedArrayUtils.java

```diff
@@ -0,0 +1,90 @@
+package org.apache.commons.lang3;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.function.Function;
+
+
+/**
+ * Operations on sorted arrays.
+ *
+ * @author Zbynek Vyskovsky
```

Comment:
We do not use `@author` tags, please remove it. Contributors are usually credited in `changes.xml` after a PR is merged.

Reply:
Done.  I noticed it somewhere else seems legacy.

Reply:
It just dawned on me that this new method would belong in the existing 'ArrayUtils' class.

Reply:
@garydgregory : I thought about it originally but:
- `ArrayUtils` already contains quite a lot of functions and may become too messy.  I agree it may not be a valid argument though.
- These functions require special semantics from the arrays (be sorted), so I thought it would be better to explicitly name the wrapping class.

I'm open to move them to `ArrayUtils` though, if you don't agree.

Reply:
Hello @kvr000 
This only makes sense in the 'ArrayUtils' class IMO.